tycho-common/src/storage.rs

```diff
@@ -478,6 +478,23 @@ pub trait ProtocolGateway {
         chain: &Chain,
         pagination_params: Option<&PaginationParams>,
     ) -> Result<WithTotal<Vec<String>>, StorageError>;
+
+    /// Retrieve the total value locked for a component
+    ///
+    /// Fetches the total value locked for a component
```

Comment:
Considering it's getting multiple tvls we should adjust this doc string. Also the second sentence doesn't add new info so we could skip it.
```suggestion
    /// Retrieve the components total value locked (TVL).
```